src/adapter/src/catalog/builtin.rs

```diff
@@ -5072,6 +5102,7 @@ pub static BUILTINS_STATIC: Lazy<Vec<Builtin<NameReference>>> = Lazy::new(|| {
         Builtin::Log(&MZ_COMPUTE_FRONTIERS_PER_WORKER),
         Builtin::Log(&MZ_COMPUTE_IMPORT_FRONTIERS_PER_WORKER),
         Builtin::Log(&MZ_COMPUTE_DELAYS_HISTOGRAM_RAW),
+        Builtin::Log(&MZ_COMPUTE_ERROR_COUNTS_RAW),
```

Comment:
I think you're missing `MZ_ARRANGEMENT_RECORDS_RAW`.

Reply:
That already exists, Jan just moved it around